File: 0201_0300/0288/0288.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = f.read()
    n = len(s)
    i, k = 0, 0
    while i < n:
        if s[i] == "'":
            i = s.index("'", i + 1)
        elif s[i : i + 2] == '//':
            k += 1
            i = s.index('\n', i + 2) if '\n' in s[i + 2 : ] else n - 1
        elif s[i : i + 2] == '(*':
            k += 1
            i = s.index('*)', i + 2)
        elif s[i] == '{':
            k += 1
            i = s.index('}', i + 1)
        i += 1
    q.write(str(k))

File: 0201_0300/0288/0288.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int k = 0;
	char s[16385], *t;

	s[fread(s, 1, 16384, f)] = '\0';
	t = s;
	while (*t)
	{
		switch (*t)
		{
		case '\'':
			while (*++t != '\'');
			break;
		case '{':
			while (*++t != '}');
			++k;
			break;
		case '(':
			if (*(t + 1) == '*' && (t += 3) && ++k)
				while (!(*t == ')' && *(t - 1) == '*'))
					++t;
			break;
		case '/':
			if (*(t + 1) == '/' && ++k)
				while (*++t != '\n' && *(t + 1));
			break;
		}
		++t;
	}
	fprintf(q, "%d", k);
	return 0;
}
